src/main/java/org/jetbrains/id/names/suggesting/contributors/NGramVariableNamesContributor.java

```diff
@@ -59,6 +75,22 @@ private static boolean isSupported(@NotNull PsiNameIdentifierOwner identifierOwn
                 .stream();
     }
 
+    public static Boolean caretInsideVariable(@NotNull PsiVariable variable) {
```

Comment:
Not sure if `NGramVariableNamesContributor.java` is the best place for the method. `PsiUtils` suits much better